VitruvioHost/Plugins/Vitruvio/Extras/UnrealGeometryEncoder/Private/Codec/Encoder/UnrealGeometryEncoder.cpp

```diff
@@ -682,14 +667,36 @@ void UnrealGeometryEncoder::convertGeometry(const prtx::InitialShape& initialSha
 	if (geometries.size() > 0)
 	{
 		const SerializedGeometry sg = serializeGeometry(geometries, materials);
-		encodeMesh(cb, sg, initialShape.getName(), L"", prtx::EncodePreparator::FinalizedInstance::NO_PROTOTYPE_INDEX, L"", geometries, materials);
+		encodeMesh(cb, sg, L"", L"", prtx::EncodePreparator::FinalizedInstance::NO_PROTOTYPE_INDEX, L"", geometries, materials);
```

Comment:
The `convertGeometry` function is now called form the `finish` callback where we don't have access to the initial shape anymore. Also because we can now encode several initial shapes for the batch generate, so I left the names empty for the generated mesh. It is still used for instances though.